src/widgets/barchart/mod.rs

```diff
@@ -8,14 +9,28 @@ pub use bar_group::BarGroup;
 
 use super::{Block, Widget};
 
-/// Display multiple bars in a single widgets
+/// A chart showing values as [bars](Bar).
```

Comment:
Does it make sense to show somewhere a complete group chart like the following:
```
┌─────────────────────────────────┐
│                             ████│
│                             ████│
│                        ▅▅▅▅ ████│
│            ▇▇▇▇        ████ ████│
│            ████ ████   ████ ████│
│     ▄▄▄▄   ████ ████   ████ ████│
│▆10▆ 20M█   █50█ █40█   █60█ █90█│
│ C1   C2     C1   C2     C1   C2 │
│ Group1      Group2      Group3  │
└─────────────────────────────────┘
```
So that the reader knows directly, what is possible to do.
(I am not sure if this is the right place)

Reply:
I think either a textual chart or a VHS is useful.

I can add a textual representation now and we'll decide later if we replace them with VHS.

Reply:
Added in 09c974bce45c368ba7408c8efa729db324dec795.

I basically took your example and just made it a bit smaller.

<details><summary>Screenshot</summary>
<p>

![image](https://github.com/ratatui-org/ratatui/assets/36198422/75f29387-7cc1-4345-9200-d24fb96c706d)

</p>
</details>